Helper_Scripts/Evals/ingest_rag_eval_workflow.py

```
@@ -37,8 +37,37 @@
 from dataclasses import dataclass
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Tuple
+from urllib.parse import urlparse
 
-import requests
+
+def _ensure_repo_root() -> None:
+    here = Path(__file__).resolve()
+    for parent in [here] + list(here.parents):
```

Ruff (0.14.10)

Consider `[here, *list(here.parents)]` instead of concatenation

Replace with `[here, *list(here.parents)]`

(RUF005)